metadata/InvoicesIndia__1__metadata.json

```diff
@@ -2,7 +2,7 @@
   "changeLog": "",
   "cpu": 0,
   "description": "Machine Learning model for extracting commonly occurring data points from Indian Invoices, including header fields and line items. Please see more details including supported languages and link to Activities guide in the About Licensing -> Document Understanding API Key section of the UiPath Automation Cloud Guide here: https://docs.uipath.com/automation-cloud/docs/about-licensing#section-document-understanding-api-key.",
-  "displayName": "IndianInvoices",
+  "displayName": "InvoicesIndia",
```

Comment:
"name" and "displayName" should match, "name" also needs to change to "InvoicesIndia"